CharlieBackend.Business/Services/CourseService.cs

```diff
@@ -52,8 +53,9 @@ public async Task<Result<CourseDto>> CreateCourseAsync(CreateCourseDto courseDto
 
         public async Task<Result<IList<CourseDto>>> GetAllCoursesAsync()
```

Comment:
there's no point in using Result for list methods (since there's no validation happens)